pkg/virt-controller/services/template_test.go

```diff
@@ -3495,18 +3495,12 @@ var _ = Describe("Template", func() {
 		},
 			Entry("on a root virt-launcher", func() *v1.VirtualMachineInstance {
 				return api.NewMinimalVMI("fake-vmi")
-			}, &kubev1.PodSecurityContext{
-				RunAsUser: new(int64),
-			}),
+			}, &kubev1.PodSecurityContext{}),
 			Entry("on a non-root virt-launcher", func() *v1.VirtualMachineInstance {
 				vmi := api.NewMinimalVMI("fake-vmi")
 				vmi.Status.RuntimeUser = uint64(nonRootUser)
 				return vmi
-			}, &kubev1.PodSecurityContext{
-				RunAsUser:    &nonRootUser,
-				RunAsGroup:   &nonRootUser,
-				RunAsNonRoot: pointer.Bool(true),
-			}),
+			}, &kubev1.PodSecurityContext{}),
```

Comment:
If these are now both the same security context since there is no more function to calculate the correct security context, is it necessary to have the separate cases for non-root and root virt-launcher here?

Is this table/test case even necessary at all anymore? Removing the function that calculates the security context is removing the feature that this test case is supposed to be testing which makes me feel that it is better off as a simple test rather than a table with redundant cases.

Reply:
Yes you're right the table isn't needed as the the security context can have selinux and seccomp profile and they are both tested so this table now doesn't add anything. I think it's better to remove it.